modules/testcases/jdkAll/Eap71x-Proposed/pom.xml

```diff
@@ -450,6 +450,16 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <jboss.home>${maven.multiModuleProjectDirectory}/modules/servers/eap7/build/target/jbossas</jboss.home>
```

Comment:
Is maven.multiModuleProjectDirectory set?
Could we add jboss.home system property at test-configurations (level) pom, in case in some other test configuration someone would like to set another path? (some thing similar to https://github.com/jboss-set/eap-additional-testsuite/commit/117b78e98efa412918bfeff5c0620c72527a1cf4)

Reply:
maven.multiModuleProjectDirectory is (should be) set by maven.

You can always override jboss.home in a submodule configuration. I'd rather leave this on this level as a reasonable default value and override at test-configuration level if needed. WDYT?